morpheus-graphql-client/src/Data/Morpheus/Client/Fetch/WebSockets.hs

```diff
@@ -101,19 +102,19 @@ decodeMessage :: A.FromJSON a => ByteString -> GQLClientResult a
 decodeMessage = (first FetchErrorParseFailure . A.eitherDecode) >=> processMessage
 
 initialMessage :: ApolloSubscription ()
-initialMessage = ApolloSubscription {apolloType = "connection_init", apolloPayload = Nothing, apolloId = Nothing}
+initialMessage = ApolloSubscription {apolloType = GqlConnectionInit, apolloPayload = Nothing, apolloId = Nothing}
 
 encodeRequestMessage :: (RequestType a, A.ToJSON (RequestArgs a)) => Text -> Request a -> ByteString
 encodeRequestMessage uid r =
   A.encode
     ApolloSubscription
       { apolloPayload = Just (toRequest r),
-        apolloType = "start",
+        apolloType = GqlSubscribe,
         apolloId = Just uid
       }
 
 endMessage :: Text -> ApolloSubscription ()
```

Comment:
will this still support old protocols? since some of the m are working with "start" and "stop". may we can adjust it depending which protocol client will use?

Reply:
As it is, unfortunately not. Renaming the types to suit the newer protocol breaks all the others.

I'm not sure how we would update the code to support multiple protocols - possibly some ReaderT instance but that is too complicated for my skills to integrate into the current codebase I think.

Reply:
would it be then okey. if i take over the pr?

Reply:
will crate followup PR for that